ui/components/requirements.jsx

```diff
@@ -66,6 +73,9 @@ Requirements.propTypes = {
   keywords: FilterList.propTypes.existingFilters,
   pagedReqs: React.PropTypes.shape({
     results: React.PropTypes.arrayOf(React.PropTypes.shape({
+      policy: React.PropTypes.shape({
+        sunset: React.PropTypes.string,
+      }),
```

Comment:
If we remove the `sunset` attribute, I _think_ we can remove this type check, too. It might make sense to just reference `Requirement.propTypes.requirement` (would require re-ordering the file, though).